src/assets/less/style.less

```diff
@@ -270,6 +275,43 @@ body {
       }
     }
 
+    // Game header
+    header.top {
+      position: relative;
+      margin: -@spacing__xxl*2 calc(@spacing__m*-1) 0;
+      padding: @spacing__xxl*2 @spacing__m @spacing__xl;
+      width: calc(100% + @spacing__m*2);
+      z-index: 3;
+      
+      h1 {
+        margin-top: @spacing__base;
+        color: @white;
+      }
+    }
+    
+    .game.sing-together {
+      
+      header.top {
+        background:
+        var(--gameHeaderDecorationSharp) center,
+        linear-gradient(265deg, rgba(28, 26, 33, 0.5) 0%, rgba(43, 37, 7, 0.75) 100%);
+        background-size: cover;
+        background-position-y: 57%;
+      }
+    }
+    
+    .game.trivia {
+      
+      header.top {
```

Comment:
We could merge this with .game.trivia above, if there is no further styling needed.
But thats only readability.

Reply:
background-size is the only one that they have in common? In what way would you merge them?